framework/src/main/java/org/checkerframework/framework/stub/AnnotationFileElementTypes.java

```diff
@@ -201,25 +201,26 @@ public void parseStubFiles() {
                             annotationFileStream,
                             factory,
                             processingEnv,
-                            stubAnnos);
+                            annotationFileAnnos);
                 }
             } else {
-                // We didn't find the stub files.
-                // If the stub file has a prefix of "checker.jar/" then look for the file in the top
+                // We didn't find the files.
```

Comment:
This is in a method `parseStubFiles`, so saying `stub` here makes sense. Same comment applies to rest of method.